pkg/common/helper/helper.go

```diff
@@ -340,3 +341,14 @@ func (h *H) InspectState() {
 	err = h.inspect(h.CurrentProject())
 	Expect(err).ShouldNot(HaveOccurred(), "could not inspect project '%s'", h.proj)
 }
+
+// GetClusterVersion returns the Cluster Version object and current cluster version ex. 4.7.1 as string
+func (h *H) GetClusterVersion() (*configv1.ClusterVersion, string, error) {
+	cfgClient := h.Cfg()
+	getOpts := metav1.GetOptions{}
+	clusterVersionObj, err := cfgClient.ConfigV1().ClusterVersions().Get(context.TODO(), "version", getOpts)
+	if err != nil {
+		return nil, "", fmt.Errorf("couldn't get current ClusterVersion '%s': %v", "version", err)
+	}
+	return clusterVersionObj, clusterVersionObj.Status.Desired.Version, nil
```

Comment:
I think this might be kinda pedantic and awful of me, but I don't think I understand why it's worth adding another return value here when you're already returning the struct containing that value as the first return value. I can kinda see that the string of nested field accesses is long and it saves you from doing that in other code, but I'm not sure that I think the tradeoff is worth it.

I won't block on this, but I'd recommend dropping back to two return values.

Reply:
Fair enough, keeping the business logic down stream makes sense due to unintended increases in time complexity. I will adjust :)